
From: Aneesh Kumar <aneesh.kumar@gmail.com>

The below patch add the symbol information of the pc and ra to the Oops
message.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/arch/alpha/kernel/traps.c |    2 ++
 1 files changed, 2 insertions(+)

diff -puN arch/alpha/kernel/traps.c~alpha-print-the-symbol-of-pc-and-ra-during-oops arch/alpha/kernel/traps.c
--- 25/arch/alpha/kernel/traps.c~alpha-print-the-symbol-of-pc-and-ra-during-oops	2004-07-31 17:26:43.785216136 -0700
+++ 25-akpm/arch/alpha/kernel/traps.c	2004-07-31 17:26:43.789215528 -0700
@@ -65,6 +65,8 @@ dik_show_regs(struct pt_regs *regs, unsi
 {
 	printk("pc = [<%016lx>]  ra = [<%016lx>]  ps = %04lx    %s\n",
 	       regs->pc, regs->r26, regs->ps, print_tainted());
+	print_symbol("pc is at %s\n", regs->pc);
+	print_symbol("ra is at %s\n", regs->r26 );
 	printk("v0 = %016lx  t0 = %016lx  t1 = %016lx\n",
 	       regs->r0, regs->r1, regs->r2);
 	printk("t2 = %016lx  t3 = %016lx  t4 = %016lx\n",
_
